CONTRIBUTING.md

```diff
@@ -1,53 +1,102 @@
 
-The main conventions you'll need to be aware of are as follows:
+The codebase follows certain conventions (including some inherited
+from the [Project Liberty Contribution
+Standards](https://github.com/LibertyDSNP/liberty-web/blob/main/CONTRIBUTING.md).
+Many of the style conventions are enforced by the automated linters so
+that you don't have to spend hours going over every line to make sure
+it matches all conventions.  Many IDEs can run these linters
+automatically, but if yours doesn't, you can run them manually with
+'npm run format'.
+
+The main conventions you need to know about are:
+
 * All code written in Typescript with as much type specificity as possible.
+
 * Styling for visual components. [Check the Style Guide](https://github.com/LibertyDSNP/example-client/blob/main/STYLING.md)
-* Use of the [BEM](https://en.bem.info/methodology/quick-start/) methodolgy for CSS classes
+
+* Use of the [BEM](https://en.bem.info/methodology/quick-start/) methodology for CSS classes
+
 * Use of [React Hooks](https://reactjs.org/docs/hooks-intro.html) in components
+
 * Testing of all added components using [Jest](https://jestjs.io) and [Enzyme](https://enzymejs.github.io/enzyme/).
 
 ### Testing
-As mentioned above testing is a very important part of this project. All code aditions must have tests that cover their use cases. Tests do not need to be exhaustive, but a good covering of main functionality is expected. It is highly recommeneded to view some of the already written tests to use as examples. You can run tests locally with 'npm run test'.
+
+All code additions must have tests that cover their use cases.  Tests
+do not need to be exhaustive, but a good covering of main
+functionality is expected.  We recommend viewing some of the
+already-written tests as examples.
+
+Run tests locally with 'npm run test'.
 
 ### Pull Requests
-Once you have written your contribution and it matches all conventions and has proper testing in place you can create a Pull Request on the Github repository. All PRs must follow the `Pull Request Template`. To be approved and merged there are some specific requirements that must be met:
+
+All PRs must follow the `Pull Request` template.  To be approved and
+merged, a PR must meet these requirements:
+
 1. The test must pass CI/CD done through Github Actions
-2. A member of the DSNP team must approve the changes
-3. The changes must be rebased on top of the main branch before merging.
+2. The changes must be rebased on the `main` branch before merging.
```

Comment:
Periods or no periods at the end of lists?  I vote full sentences and periods, but I think we need to have them all follow whatever convention we land on.

Reply:
+1 to what @claireolmstead says above, FWIW (but I don't feel strongly either way).